components/EmailValidator.vue

```diff
@@ -6,18 +6,20 @@
       label-class="mt-0"
       :state="true"
     >
-      <Field
-        ref="email"
-        v-model="currentEmail"
-        :rules="validateEmail"
-        type="email"
-        name="email"
-        :class="'email form-control input-' + size + ' ' + inputClass"
-        :center="center"
-        autocomplete="username email"
-        :placeholder="'Email address ' + (required ? '' : '(Optional)')"
-      />
-      <ErrorMessage name="email" class="text-danger font-weight-bold" />
+      <Form ref="form">
```

Comment:
Have you checked that this component is never used within another `form`?  I don't think nested forms are valid.

I don't know whether it is - I've not checked.

Reply:
We need to have <Form /> wrapper for ErrorMessage to work.
https://vee-validate.logaretm.com/v4/api/error-message/#caveats

It seems we can render vee-validate <Form /> renderless. I'll push the fix. But we probably need to rework vee-validate usage and move to their composables. It should help to integrate between bootstrap components and vee-validate validation logic.

Reply:
Have you pushed this change?  I can't see it but I'm not sure what I'm looking for.

We don't use vee-validate much, because it's felt clunky.  I'd be interested to see if the composables feel better.  Please open a PR for that.

Reply:
Yeah, it's there. <Form as="">